.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
   - TOXENV=pypy
   - TOXENV=py33
   - TOXENV=py34
+  - TOXENV=py35
```

Comment:
To make it work use python:3.5 instead of python:2.7, Travis will download Python 3.5 in this case.